components/calendar/README.md

```diff
@@ -1,21 +1,26 @@
-# Calendar
+# Calendars
 
 The `d2l-calendar` component can be used to display a responsively sized calendar that allows for date selection. It indicates the currently selected date if `selected-value` is specified, or if the user selects a date.
 
+<!-- docs: start hidden content -->
 ![Calendar](./screenshots/calendar.png?raw=true)
+<!-- docs: end hidden content -->
 
+## Calendar [d2l-calendar]
+
+<!-- docs: demo live name:d2l-calendar -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/calendar/calendar.js';
 </script>
-<!-- Today is February 21, 2020 -->
 <d2l-calendar
 	selected-value="2020-02-09"
 	summary="Click on a day to select it as the assignment due date.">
 </d2l-calendar>
 ```
 
-**Properties:**
+<!-- docs: start hidden content -->
```

Comment:
No closing tag for this hidden content

Reply:
There's one below on line 37, just after events